.github/workflows/update-minimum-android-and-ios-sdk.yml

```diff
@@ -0,0 +1,62 @@
+name: Update Minimum Android and iOS SDK
+
+on:
+  workflow_dispatch:
+    inputs:
+      android_minimum_version:
+        description: 'Android SDK Minimum Version'
+        required: false
+      ios_minimum_version:
+        description: 'iOS SDK Minimum Version'
+        required: false
+
+jobs:
+  update-versions:
+    name: Update Minimum Android and iOS SDK
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout source branch
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Change Git Branch
+        run: |
+          git checkout -b fix/updateMinimumSdkVersions
```

Comment:
how do you know that this branch has the lastest code? I suggest creating the branch from latest master (like hotfix action does) and add the date in the branch name to avoid clashes

Reply:
I've addressed this except for creating the branch from the latest master: I don't want to restrict this GitHub Action to the master branch only. I've envisioned using this GitHub Action from any branch.
If you still feel that I should restrict it to master please LMK once more.